tfx_addons/version.py

```diff
@@ -68,4 +68,5 @@
         "slackclient>=2.9.0,<3.0",
         "pydantic>=1.8.0,<2.0",
     ],
+    "pandas_transform": [f"tfx{_TFXVERSION_CONSTRAINT}", "pandas>=1.0.0"],
```

Comment:
nit: Should we add a cap to pandas? (aka <2.0) to avoid breaking if there's a new major version of pandas?

Reply:
Good idea, added.